### PR TITLE
RUMM-1956 Document all regions in setup instruction

### PR DESCRIPTION
**Pull request by maxep**
### What and why?

All current regions (US1, US3, US5, US1_FED, EU1)  are documented in the setup instructions.

This PR also fix tracing indentation and RUM setup for eu1.

### Review checklist

- [x] Make sure each commit and the PR mention the Issue number or JIRA reference

### CODE REVIEW

**Comment by alai97** [comment]
Created test preview branch in [#13052](https://github.com/DataDog/documentation/pull/13052)!

File: docs/trace_collection.md [hunk 1]
```diff
@@ -105,20 +105,110 @@ DDConfigurationBuilder *builder = [DDConfiguration builderWithClientToken:@"<cli
...
+[DDDatadog initializeWithAppContext:[DDAppContext new]
+                    trackingConsent:trackingConsent
+                      configuration:[builder build]];
+```
+{{% /tab %}}
+{{< /tabs >}}
+{{< /site-region >}}
+
+To be compliant with the GDPR regulation, the SDK requires the `trackingConsent` value at initialization.
+The `trackingConsent` can be one of the following values:
```
Comment (alai97) [inline docs/trace_collection.md:199]:
```suggestion
To comply with GDPR regulations, the SDK requires the `trackingConsent` value at initialization.

Use one of the following values for `trackingConsent`:
```

File: docs/trace_collection.md [hunk 2]
```diff
@@ -105,20 +105,110 @@ DDConfigurationBuilder *builder = [DDConfiguration builderWithClientToken:@"<cli
...
 
-    - `.pending` - the SDK starts collecting and batching the data but does not send it to Datadog. The SDK waits for the new tracking consent value to decide what to do with the batched data.
-    - `.granted` - the SDK starts collecting the data and sends it to Datadog.
-    - `.notGranted` - the SDK does not collect any data: logs, traces, and RUM events are not sent to Datadog.
+- `.pending` - the SDK starts collecting and batching the data but does not send it to Datadog. The SDK waits for the new tracking consent value to decide what to do with the batched data.
+- `.granted` - the SDK starts collecting the data and sends it to Datadog.
+- `.notGranted` - the SDK does not collect any data: logs, traces, and RUM events are not sent to Datadog.
```
Comment (alai97) [inline docs/trace_collection.md:203]:
```suggestion
- `.pending` - the SDK starts collecting and batching the data but does not send it to Datadog. The SDK waits for the new tracking consent value to decide what to do with the batched data.
- `.granted` - the SDK starts collecting the data and sends it to Datadog.
- `.notGranted` - the SDK does not collect any data. No logs, traces, or RUM events are sent to Datadog.
```

File: docs/trace_collection.md [hunk 3]
```diff
@@ -105,20 +105,110 @@ DDConfigurationBuilder *builder = [DDConfiguration builderWithClientToken:@"<cli
...
 
-    To change the tracking consent value after the SDK is initialized, use the `Datadog.set(trackingConsent:)` API call.
-    The SDK changes its behavior according to the new value. For example, if the current tracking consent is `.pending`:
+To change the tracking consent value after the SDK is initialized, use the `Datadog.set(trackingConsent:)` API call.
+The SDK changes its behavior according to the new value. For example, if the current tracking consent is `.pending`:
 
-    - if changed to `.granted`, the SDK will send all current and future data to Datadog;
-    - if changed to `.notGranted`, the SDK will wipe all current data and will not collect any future data.
+- if changed to `.granted`, the SDK will send all current and future data to Datadog;
+- if changed to `.notGranted`, the SDK will wipe all current data and will not collect any future data.
```
Comment (alai97) [inline docs/trace_collection.md:209]:
```suggestion
- If changed to `.granted`, the SDK sends all current and future data to Datadog.
- If changed to `.notGranted`, the SDK wipes all current data and does not collect future data.
```

**Review by alai97 — APPROVED** [review]
Looks good, small copy nits!

**Comment by maxep** [comment]
Thanks for the branch @alai97! It looks like it's working great!

**Review by ncreated — APPROVED** [review]
👌

**Pull request merged**